.github/workflows/ci_cd.yml

```diff
@@ -15,12 +15,15 @@ jobs:
     outputs:
       # The packages that use Uniffi bindings
       ffi_packages: ${{ steps.set_ffi_packages.outputs.ffi_packages }}
+      # The API client packages
+      api_packages: ${{ steps.set_api_packages.outputs.api_packages }}
     steps:
       - id: set_ffi_packages
         run: echo 'ffi_packages=["algokit_transact"]' >> $GITHUB_OUTPUT
+      - id: set_api_packages
+        run: echo 'api_packages=["algod_api"]' >> $GITHUB_OUTPUT
 
   typescript_wasm_ci_cd:
-    concurrency: ${{ github.event_name == 'push' && format('cd-{0}', github.ref) || format('ci-typescript_wasm-{0}', github.run_id) }}
```

Comment:
I'm assuming this was added as a mechanism for running these steps sequentially only when running release builds?

Unfortunately concurrency in GH actions doesn't quite work like that and instead returns `Canceling since a higher priority waiting request for '{build}' exists`.